src/audio/nxp/eap_stub.c

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2025 NXP
+
+#include <sof/audio/nxp/eap/EAP_Includes/EAP16.h>
+
+LVM_ReturnStatus_en LVM_GetVersionInfo(LVM_VersionInfo_st  *pVersion)
```

Comment:
Excellent, thanks for adding the stub! This is really useful for upstream testing and CI to be able to do basic build tests.